upgrade_test.py

```diff
@@ -479,6 +484,12 @@ def get_highest_supported_sstable_version(self):  # pylint: disable=invalid-name
         output = []
         for node in self.db_cluster.nodes:
             output.extend(get_node_supported_sstable_versions(node.system_log))
+
+        if not output:
+            for node in self.db_cluster.nodes:
+                with self.db_cluster.cql_connection_patient_exclusive(node) as session:
+                    output.extend(get_node_enabled_sstable_version(session))
```

Comment:
I don't understand this code ?

why not replacing this one with the previous ?

Reply:
I renamed the variable to be more meaningful. and also i don't remove old code, which use logs, because  master branch is also used with different version, where old functionality is used.
So if sstable_format was not parsed from logs (as in old versions) then it will be get from scylla table for new versions

Reply:
reading it from logs doesn't works for new versions  ?
reading it from system tables doesn't work for old versions ?

I'm a bit a lot, what exactly happened that we need to touch any of those parts

if we do, I would want to know why, and make sure we don't keep duplication for no real reason.